spec/ace.md

```diff
@@ -49,6 +51,14 @@ Volumes that are specified in the Pod Manifest are mounted into each of the apps
 For example, say that the worker-backup and reduce-worker both have a `mountPoint` named "work".
 In this case, the executor will bind mount the host's `/opt/tenant1/work` directory into the `path` of each of the matching "work" `mountPoint`s of the two app filesystems.
 
+If the target `path` does not exist in the rendered filesystem, it SHOULD be created, including any missing parent directories. If the target `path` is a non-empty directory, its contents SHOULD be discarded (e.g. obscured by the bind mount). If the target `path` refers to a file, ACE SHOULD remove that file and create a directory in its place.
+
+Mount point `path` directories that ACE creates SHOULD be owned by UID 0 and GID 0, and have access mode `0755` (`rwxr-xr-xr-x`). ACE implementation MAY provide a method for administrator to specify different permissions on a per-pod basis.
```

Comment:
> flexibility of mounting volumes anywhere via app overrides seems to be a feature rather than a bug. 

Related, would appreciate your input/proposal for https://github.com/appc/spec/issues/364